business/jaeger_helper.go

```diff
@@ -29,7 +29,7 @@ type JaegerServices struct {
 func getErrorTracesFromJaeger(namespace string, service string) (errorTraces int, err error) {
 	errorTraces = 0
 	err = nil
-	if !appstate.JaegerAvailable || !config.Get().ExternalServices.Tracing.Enabled {
+	if !config.Get().ExternalServices.Tracing.Enabled {
 		return -1, errors.New("jaeger is not available")
 	}
 	if appstate.JaegerEnabled {
```

Comment:
But now we can't check if the problem is in Jaeger or in Kiali.

config.Get().ExternalServices.Tracing.Enabled // User enable configuration
JaegerEnabled  // Kiali discover jaeger o check that the information provided by user is ok
JaegerAvailable // Jaeger application is not available.

Reply:
I'm not sure to understand. When jaeger is not available, we get an error message in Kiali. So the user will know that something is wrong with Jaeger.

Reply:
The point was to check if Jaeger was ok but the application is failling for example by elastic search, but maybe we can improve this in Kiali 2.0 :)

Reply:
We can check this scenario with jaeger-query up and elastic down, I'm not sure how the error will be reported / maybe will have a timeout. However, either way I think an error should still be caught at fetch time rather than at Kiali startup.

Reply:
So, after checking with @hhovsepy it's basically the same from Kiali standpoint to say elastic is failing, or Jaeger is failing. The scenario with failing Jaeger has been tested here and seems OK, as expected.